packages/react-context/tsconfig.json

```diff
@@ -4,5 +4,6 @@
     "rootDir": "src",
     "outDir": "build"
   },
-  "include": ["src"]
+  "include": ["src"],
+  "references": [{ "path": "../auth" }]
```

Comment:
Can we infer this from dependencies?

Reply:
#23 actually kind of implemented that, but only checks that they are there. TypeScript inferring them automatically is unfortunately still not possible https://github.com/microsoft/TypeScript/issues/25376